### PR TITLE
Fixing markdown errors in the repo 

### PR DESCRIPTION
**Pull request by kshithijiyer**

### CODE REVIEW

**Comment by kshithijiyer** [comment]
@harigowtham Could you please review this pull request?

File: docs/Administrator Guide/index.md [hunk 1]
```diff
@@ -58,7 +56,7 @@
 
 17.  [Trash for GlusterFS](./Trash.md)
 
-18.  [Split brain and ways to deal with it](Split brain and ways to deal with it.md)
+18.  [Split brain and ways to deal with it](./Split brain and ways to deal with it.md)
```
Comment (harigowtham) [inline docs/Administrator Guide/index.md]: There are a few places like this where the link is not pointing to the file. The file might be at a different location or unavailable. Can this be looked into?

File: docs/Administrator Guide/Setting Up Clients.md [hunk 2]
```diff
@@ -80,12 +80,8 @@ To install Gluster Native Client on RPM distribution-based systems
     `$ sudo rpm -i glusterfs-fuse-3.8.5-1.x86_64`
     `$ sudo rpm -i glusterfs-rdma-3.8.5-1.x86_64`
 ~~~
-
-```
- **Note**
-
-     The RDMA module is only required when using Infiniband.
-```
+> **Note:**
```
[the remark this answers is not in the document]
Comment (kshithijiyer) [reply docs/Administrator Guide/Setting Up Clients.md]: Done

File: docs/Administrator Guide/GlusterFS iSCSI.md [hunk 3]
```diff
@@ -1,3 +1,6 @@
+GlusterFS iSCSI
```
[the remark this answers is not in the document]
Comment (kshithijiyer) [reply docs/Administrator Guide/GlusterFS iSCSI.md]: Done

File: docs/Administrator Guide/Logging.md [hunk 4]
```diff
@@ -3,54 +3,54 @@
 Below lists the component, services, and functionality based logs in the GlusterFS Server. As per the File System Hierarchy Standards (FHS) all the log files are placed in the `/var/log` directory.
 ⁠
 
-##glusterd:
+## glusterd:
```
[the remark this answers is not in the document]
Comment (kshithijiyer) [reply docs/Administrator Guide/Logging.md]: Done

File: docs/Administrator Guide/index.md (hunk 1 above)
Comment (kshithijiyer) [reply docs/Administrator Guide/index.md]: I am aware of this have ran **markdown-link-check** on the entire repo will put up a new pull request to address this issues as it'll take some good amount of work. Would that be okay?

File: docs/Administrator Guide/Directory Quota.md [hunk 5]
```diff
@@ -64,25 +66,25 @@ being treated as "/".
 **To set or replace disk limit:**
 
 -   Set the disk limit using the following command:
+```
```
[the remark this answers is not in the document]
Comment (kshithijiyer) [reply docs/Administrator Guide/Directory Quota.md]: You are correct have reverted back to original formatting.

**Comment by kshithijiyer** [comment]
@harigowtham anything else is to be changed?

File: docs/Administrator Guide/GlusterFS Cinder.md [hunk 6]
```diff
@@ -87,8 +78,7 @@ Restart the firewall service:
 
 		$ sudo service iptables restart
 
-7. Restarting Cinder Services
-=============================
+## 7. Restarting Cinder Services
```
[the remark this answers is not in the document]
Comment (harigowtham) [reply docs/Administrator Guide/GlusterFS Cinder.md]: I still find these to be in italic. Can this be changed to normal text?

File: docs/Administrator Guide/GlusterFS Cinder.md [hunk 7]
```diff
@@ -98,8 +88,7 @@ Check the Cinder volume log to make sure that there are no errors:
 
 		$ sudo tail -50 /var/log/cinder/volume.log
 
-8. Verify GlusterFS Integration with Cinder
-===========================================
+## 8. Verify GlusterFS Integration with Cinder
```
[the remark this answers is not in the document]
Comment (harigowtham) [reply docs/Administrator Guide/GlusterFS Cinder.md]: same as above

File: docs/Administrator Guide/GlusterFS iSCSI.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (harigowtham) [reply docs/Administrator Guide/GlusterFS iSCSI.md]: This change is pending as well. 
Use it more like 
# GlusterFS iSCSI

## Introduction

The lines are removed and the '#' are used to format the page.

File: docs/Administrator Guide/Directory Quota.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (harigowtham) [reply docs/Administrator Guide/Directory Quota.md]: Thanks

File: docs/Administrator Guide/GlusterFS iSCSI.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (harigowtham) [reply docs/Administrator Guide/GlusterFS iSCSI.md]: Sorry, checked an outdated patchset. This is fixed.

File: docs/Administrator Guide/index.md (hunk 1 above)
Comment (harigowtham) [reply docs/Administrator Guide/index.md]: Yep, you can address that as a separate PR.

File: docs/Administrator Guide/GlusterFS Cinder.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (kshithijiyer) [reply docs/Administrator Guide/GlusterFS Cinder.md]: Done

File: docs/Administrator Guide/index.md (hunk 1 above)
Comment (kshithijiyer) [reply docs/Administrator Guide/index.md]: PR sent: https://github.com/gluster/glusterdocs/pull/505

**Pull request merged**